prog/download_boot_image.rb

```diff
@@ -26,9 +26,9 @@ def url
       elsif image_name == "ubuntu-jammy"
         arch = (vm_host.arch == "x64") ? "amd64" : "arm64"
         "https://cloud-images.ubuntu.com/releases/jammy/release-#{version}/ubuntu-22.04-server-cloudimg-#{arch}.img"
-      elsif image_name == "almalinux-9.3"
+      elsif image_name == "almalinux-9"
         arch = (vm_host.arch == "x64") ? "x86_64" : "aarch64"
-        "https://vault.almalinux.org/9.3/cloud/#{arch}/images/AlmaLinux-9-GenericCloud-9.3-#{version}.#{arch}.qcow2"
+        "https://repo.almalinux.org/almalinux/9/cloud/#{arch}/images/AlmaLinux-9-GenericCloud-#{version}.#{arch}.qcow2"
```

Comment:
They delete old images from `https://repo.almaunix.org/`, so this will give 404 after their next release. Not sure if this is okay or not. However, they keep the images in `https://valut.almalinux.org/` for longer.

Reply:
Looks like new versions are not in vault. I complained a bit about this making permalinks difficult, but between "lagging a version behind" and "need to figure out something to deal with this archiving problem," perhaps we should do "need to figure something out" (or change links to vault as they age out)